enterprise/frontend/src/embedding-sdk/cli/steps/setup-license.ts

```diff
@@ -0,0 +1,68 @@
+import { input } from "@inquirer/prompts";
+import chalk from "chalk";
+import toggle from "inquirer-toggle";
+import open from "open";
+
+import type { CliStepMethod } from "embedding-sdk/cli/types/cli";
+
+import { printEmptyLines, printInfo } from "../utils/print";
+import { propagateErrorResponse } from "../utils/propagate-error-response";
+
+const trialUrl = `https://store.metabase.com/checkout?plan=pro&deployment=self-hosted`;
+const trialUrlWithUtm = `${trialUrl}&utm_source=product&utm_medium=checkout&utm_campaign=embedding-sdk&utm_content=embedding-sdk-cli`;
+
+export const setupLicense: CliStepMethod = async state => {
+  const hasLicenseKey = await toggle({
+    message: "Do you already have a Metabase Pro license key?",
+    default: false,
+  });
+
+  if (!hasLicenseKey) {
+    printEmptyLines(1);
+    console.log(
+      `  Please sign up for a free trial of Metabase Pro self-hosted or purchase a license.`,
+    );
+
+    const shouldOpenInBrowser = await toggle({
+      message: `Open the store to get a license key?`,
+      default: true,
+    });
+
+    if (shouldOpenInBrowser) {
+      try {
+        await open(trialUrlWithUtm);
+        console.log(`  Opened ${chalk.blue(trialUrl)} in your browser.`);
```

Comment:
In case they accidentally closed the tab with the trial (or the `open` method does not work for whatever reason), they can click on the plain link instead.

Reply:
@WiNloSt yes, we intentionally omit UTM tags from the console as that would be too much noise. We only add them when we're opening the url in a browser.